app/models/service_interaction.rb

```diff
@@ -0,0 +1,2 @@
+class ServiceInteraction < ActiveRecord::Base
+end
```

Comment:
Even if we don't add the associations it'd be worth adding simple validations for what is and isn't allowed.  For example: bare minimum we probably don't want to save a `Service` without an identifier and label, but maybe there's a specific format that the label should be in, or that the identifier should be in a specific range of numbers (I'm pretty sure we'd want the presence validations, less sure about the format + range ones - but we should investigate a bit).  There's probably similar validations we could add to `Interaction` and `ServiceInteraction`.